### ISSUE TITLE
won't initialise

### ISSUE THREAD

**Issue by andrewlorenz** [codler/bullhorn-api#1]
npm i bullhorn-api

const Bullhorn = require('bullhorn-api');
const bullhorn = new Bullhorn({ .... });

fails: "TypeError: Bullhorn is not a constructor"

**Comment by codler** [comment]
Might depend on how you have set up your build system.
Try if this works better
```
const Bullhorn = require('bullhorn-api').default;

**Comment by andrewlorenz** [comment]
yes thanks, that got me past the issue I raised.  But now its failing:

(node:8928) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'split' of undefined    at Bullhorn.login (node_modules\bullhorn-api\dist\index.js
:37:64)    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:8928) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:8928) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

I'm not using _anything_ other than node (v10.16.0), I'm just trying to write a server script to connect to a bullhorn service.  Thanks

**Comment by andrewlorenz** [comment]
I've worked out why this is happening - the split error occurs when then login fails (hence the redirect url isn't set and so can't be parsed). It could really fail more gracefully, as the coding just assumes the login will have worked.
